runtime/src/snapshot_utils.rs

```diff
@@ -3022,7 +3022,7 @@ mod tests {
             "Ensure rebuilding from an incremental snapshot works",
         );
         assert_eq!(
-            deserialized_bank.get_account(&key1.pubkey()),
+            deserialized_bank.get_account_modified_slot(&key1.pubkey()),
```

Comment:
I think we can simplify to assert!(deserialized_bank.get_account_modified_slot(&key1.pubkey()).is_none())

Reply:
Done!